doc/source/values.rst

```diff
@@ -0,0 +1,80 @@
+.. _values:
+
+Our mission
+-----------
+
+scikit-image aims to be the essential toolkit for scientific image analysis in
+Python. We accomplish this by:
+
+- being **easy to use**. We are careful in taking on new dependencies, and
+  sometimes cull existing ones, or make them optional.
```

Comment:
- promoting education and self-training on image processing, thanks to an extensive and pedagogical documentation.

Reply:
added!